### ISSUE TITLE
Usage limit indicator and wait timer for Colab Pro and Pro+

### ISSUE THREAD

**Issue by rs3dtouch** [googlecolab/colabtools#2388]
I'm using Colab Pro+ atm and reached a limit, so I can not connect to any GPU anymore. Even after 10 hours I'm off a GPU access, even the smallest GPU.

It would be nice, especially in the paid version, to have this limit indicator with a waiting timer to better manage sessions. 

Also sometimes it is enough to use a smaller GPU, which would allow more runtime overall.  So it would be helpful to choose a GPU specifically. 

Also it would be nice to have an official "End Colab Session" function, so that the running project can close the session without blocking the GPU any further. In my opinion this would be essential for an efficient use, not to block a session in idle mode.

Paying 50$ and then waiting hours, without any idea how long it will take until it is possible to work again, is frustrating...

**Comment by Snnzhao** [comment]
I also have similar problem.
My colab pro+ can access only less than 13g ram and p100 gpu.
This happens after I purchase a second colab pro+ account. Now both accounts meet this problem.
I have no idea how this happens. If google does not allow this, I can stop using the second account.
I want to know how long will this situation exists.
My accounts are 15827510651zs@gmail.com and senzhao950823@gmail.com.

**Comment by CharonWangg** [comment]
Same my account is charonwangg@gmail.com

**Comment by CyberRSR** [comment]
I have unsubscribe from Pro+ 2 days before end month, but I have active Pro+ yet, but I cannot start any more than CPU, GPU and TPU not available.

**Comment by lucids-git-goods** [comment]
Indeed.  The lack of transparency made me unsubscribe from Pro+.  I would have managed my use differently if I had any indication I'd use up my entire month's worth of time in the first couple of weeks.

**Comment by wolfgangmeyers** [comment]
This is 100% of the reason I'm not subscribing to Pro or Pro+. Buying your own hardware is more predictable.

**Comment by carriemorrison** [comment]
+1 I'm suddenly unable to connect to a GPU, there was no prior warning and isn't that the idea of paying for pro+ which is not exactly cheap. Definitely needs clearer guidance on what the plans provide and usage/threshold charts...

**Comment by qq976739120** [comment]
I have the same problem, I am a pro+ user, I can't use any gpu service, and I don't know when it will be restored

**Comment by vzakharov** [comment]
Same here with Pro+, and the lack of feedback from the provider in this thread is super-frustrating, even if somewhat expected given that that’s Google.

**Comment by iqphan** [comment]
Thanks @rs3dtouch for opening this thread. The long silence from Google Colab (after over a year) is telling... I got less than 2d of work done with my CollabPro subscription. After reading the various comments here, not only am I no longer tempted to upgrade to Pro+, but I'm actively migrating code to local resources.
